platform/marvell-armhf/sonic-platform-et6448m/et6448m_plt_setup.sh

```diff
@@ -52,6 +52,9 @@ main()
 {
     fw_uboot_env_cfg
     et6448m_profile
+
+    python /etc/entropy.py &
```

Comment:
what does this entropy.py use for?

Reply:
In A385 CPU arch, random number generator is not supported in CPU soc. During bootup, systemd requires lot of random seeds, so this has been handled in SW by providing entropy from pseudo random generator.